### PR TITLE
SH_37_light_sensor_to_mqtt____DB_saving

### PR DESCRIPTION
**Pull request by sharlatta**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/17" target="_blank">ReviewNinja

File: src/devices/deviceConnected.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        .filter(message => message.event === DEVICE_CONNECTED_EVENT)
+        .subscribe(saveDevice, onError);
+
+    function saveDevice (data) {
+        Sensor.find({description : data.value.description}, function (error, records) {
+            if (records.length){
+                console.log('Device already added'); //eslint-disable-line
+                debug('Device already added');
+            }else{
+                var device = new Sensor(data.value);
```
Comment (kosich) [inline src/devices/deviceConnected.js]: will device be inactive by default?

File: src/devices/deviceConnected.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export default function trackDeviceConnection() {
+    stream
+        .filter(message => message.event === DEVICE_CONNECTED_EVENT)
+        .subscribe(saveDevice, onError);
+
+    function saveDevice (data) {
+        Sensor.find({description : data.value.description}, function (error, records) {
+            if (records.length){
+                console.log('Device already added'); //eslint-disable-line
```
Comment (kosich) [inline src/devices/deviceConnected.js]: remove all `console.log`s

File: src/devices/deviceConnected.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+const DEVICE_CONNECTED_EVENT = 'add';
+
+export default function trackDeviceConnection() {
+    stream
+        .filter(message => message.event === DEVICE_CONNECTED_EVENT)
+        .subscribe(saveDevice, onError);
+
+    function saveDevice (data) {
+        Sensor.find({description : data.value.description}, function (error, records) {
```
Comment (kosich) [inline src/devices/deviceConnected.js]: possibly better approach with single query to the db
http://stackoverflow.com/questions/20312163/mongoose-dont-insert-if-element-already-stored
https://docs.mongodb.com/manual/reference/operator/update/setOnInsert/

**Pull request merged**